src/assets/scripts/airport/AirportController.js

```diff
@@ -188,4 +188,19 @@ export default class AirportController {
     hasAirport(icao) {
         return _has(this.airport.airports, icao);
     }
+
+    /**
+     * Remove an aircraft from the queue of any runway(s) at the AirportModel
+     * @for AirportModel
+     * @method removeAircraftFromAllRunwayQueues
+     * @param  {aircraft} aircraft The aircraft to remove
+     */
+    removeAircraftFromAllRunwayQueues(aircraft) {
+        const runways = this.airport_get().runways;
+        for (let runwayPair = 0; runwayPair < runways.length; runwayPair++) {
+            runways[runwayPair][0].removeQueue(aircraft, true);
+            runways[runwayPair][1].removeQueue(aircraft, true);
```

Comment:
ordinarily yeah.  but just at the top of the function. In this case I'd be ok with just a comment about what the index represents.